src/apis/notifications.go

```diff
@@ -0,0 +1,36 @@
+package apis
+
+import (
+	"encoding/json"
+	"os"
+
+	twilioApi "github.com/twilio/twilio-go/rest/api/v2010"
+)
+
+func (s Server) SMS(to, message string) (string, error) {
+	if err := validatePhoneNumber(to); err != nil {
+		s.Logger.Error(err)
+		return "", err
+	}
+
+	params := &twilioApi.CreateMessageParams{}
```

Comment:
Declare the ` &twilioApi.CreateMessageParams{}` inside the `fillCreateMessageParams` function